packages/rpc-core/src/rpc-methods/__tests__/get-token-accounts-by-owner-test.ts

```diff
@@ -0,0 +1,387 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
+import type { SolanaJsonRpcErrorCode } from '@solana/rpc-transport/dist/types/json-rpc-errors';
+import type { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+import fetchMock from 'jest-fetch-mock-fork';
+
+import { Commitment } from '../common';
+import { createSolanaRpcApi, SolanaRpcMethods } from '../index';
+
+// Change all these to getTokenAccountsByOwner
+// Mint fixture: fixtures/spl-token-mint-account-with-owner.json
+// Fixture: fixtures/spl-token-token-account-owner.json
+// Owner: G4QJANEpvEN8vLaaMZoWwZtqHfWxuWpd5RrVVYXPCgeL
+// Mint: 2nBoNW5B9SdpJYEg9neii7ecCJFwh6UrbXS6HFxkK7Gf
+// Program: TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA
```

Comment:
I don't think I understand what this comment is instructing

Reply:
That would be because it was a note-to-self that was never supposed to be checked in! Sorry, removed!

Reply:
Lol no prob 😂